src/NuGet/Microsoft.Orleans.Server.nuspec

```diff
@@ -18,12 +18,12 @@
     <copyright>Copyright Microsoft 2015</copyright>
     <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
     <dependencies>
-      <dependency id="ProjectOrleans.Orleans" version="1.0.0.0" />
-      <dependency id="ProjectOrleans.OrleansRuntime" version="1.0.0.0" />
-      <dependency id="ProjectOrleans.OrleansAzureUtils" version="1.0.0.0" />
-      <dependency id="ProjectOrleans.OrleansHost" version="1.0.0.0" />
-      <dependency id="ProjectOrleans.CounterControl" version="1.0.0.0" />
-      <dependency id="ProjectOrleans.OrleansProviders" version="1.0.0.0" />
+      <dependency id="Microsoft.Orleans.Core" version="1.0.0.0" />
+      <dependency id="Microsoft.Orleans.OrleansRuntime" version="1.0.0.0" />
+      <dependency id="Microsoft.Orleans.OrleansAzureUtils" version="1.0.0.0" />
+      <dependency id="Microsoft.Orleans.OrleansHost" version="1.0.0.0" />
+      <dependency id="Microsoft.Orleans.CounterControl" version="1.0.0.0" />
+      <dependency id="Microsoft.Orleans.OrleansProviders" version="1.0.0.0" />
```

Comment:
maybe also remove the second `Orleans*` here and just use `Microsoft.Orleans.Runtime`, `Microsoft.Orleans.AzureUtils`, `Microsoft.Orleans.Host`, etc. ?